linker/Linker/ILogger.cs

```diff
@@ -12,4 +12,19 @@ public interface ILogger
 	{
 		void LogMessage (MessageImportance importance, string message, params object[] values);
 	}
+
+	public class LambdaLogger : ILogger
```

Comment:
This does not make much sense to me. Why do have a double redirect? Please just implement ILogger in ILLinker the way you want